webdriver/tests/element_click/navigate.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def element_click(session, element):
+    return session.transport.send(
+        "POST", "session/{session_id}/element/{element_id}/click".format(
+            session_id=session.session_id,
+            element_id=element.id))
+
+
+def test_numbers_link(session):
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("<a href={url}>123456</a>".format(url))
+    element = session.find.css("a", all=False)
+    response = element_click(session, element)
+    assert_success(response)
+
+    assert session.url == "http://web-platform.test:8000{url}".format(url)
```

Comment:
Hard-coding the host and port in those tests still cause linter failures. So as best use what @andreastt mentioned and make us of the `server_config` or `configuration` fixture. Check the `add_cookie` tests which do similar things.